cypress/integration/create_enrollment.cy.js

```diff
@@ -0,0 +1,129 @@
+import {
+    selectProgramDimensions,
+    INPUT_ENROLLMENT,
+    selectPeriod,
+    RELATIVE,
+    getLineListTable,
+} from '../helpers/index.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const dimensionName = 'BCG doses'
+const periodLabel = 'Date of enrollment'
+
+const setUpTable = () => {
+    selectProgramDimensions({
+        inputType: INPUT_ENROLLMENT,
+        programName: 'Child Programme',
+        dimensions: [dimensionName],
+    })
+
+    selectPeriod({
```

Comment:
We're going to be doing this every time we create a new AO, which is a lot. I see the need for a one-liner function call to do this, instead of having to repeat the `periodLabel` and all the props everywhere, which would mean a lot of copy-pasting and a big risk for typos along the way. I suggest we add `dataTest` to the periods so we can click on the regardless of what they're called and just call something like `selectPeriodLast12Months` or `selectRelativePeriod(LAST_12_MONTHS)`.

Reply:
I've refactored this to use a list of `TEST_RELATIVE_PERIODS` and `TEST_FIXED_PERIODS` instead. However maybe its best to just assign these objects proper names instead (like `LAST_12_MONTHS` above) so its more straight-forward to see what's actually being used (rather than just e.g. `TEST_RELATIVE_PERIODS[0]` which has no semantic meaning.)